### PR TITLE
New Fluentd to Cloud Logging Dockerfile with args

### PR DESCRIPTION
**Pull request by satnam6502**

### CODE REVIEW

**Comment by satnam6502** [comment]
**DO NOT MERGE** 
Still testing. I think there may be issues.

**Comment by satnam6502** [comment]
OK, I've updated the config file too -- I had an incorrect path.

**Comment by satnam6502** [comment]
I've verified that this works by ingesting log lines from a synthetic logger and observing them in the Cloud Logging console. Ready for review/merge.

File: cluster/addons/fluentd-gcp/fluentd-gcp-image/Dockerfile [hunk 1]
```diff
@@ -9,12 +9,8 @@
 FROM ubuntu:14.04
 MAINTAINER Satnam Singh "satnam@google.com"
 
-# Disable prompts from apt.
-ENV DEBIAN_FRONTEND noninteractive
```
Comment (a-robinson) [inline cluster/addons/fluentd-gcp/fluentd-gcp-image/Dockerfile]: Do we not still want the DEBIAN_FRONTEND setting?

**Comment by a-robinson** [comment]
LGTM other than the minor question. Will need a quick rebase.

**Comment by satnam6502** [comment]
I found that removing DEBIAN_FRONTEND did not make any difference -- I've added it back in and re-generated the image. PTAL.

**Comment by a-robinson** [comment]
LGTM, will merge on green.

**Pull request merged**